ui/admin/app/components/form/credential-library/vault-ssh-certificate/index.hbs

```diff
@@ -26,17 +26,19 @@
     {{#if @model.errors.name}}
       <F.Error as |E|>
         {{#each @model.errors.name as |error|}}
-          <E.Message>{{error.message}}</E.Message>
+          <E.Message data-test-error-message-name>
```

Comment:
We should probably remove this `data-test` as well.

Reply:
@cameronperera ahh good catch